.github/workflows/ci.yml

```diff
@@ -170,6 +170,18 @@ jobs:
       - name: Enable corepack and pnpm
         run: corepack enable && corepack enable pnpm
 
+      - name: Tee-worker client-api install npm deps
+        working-directory: ./tee-worker/client-api
+        run: pnpm install
+
+      - name: Tee-worker client-api build
+        working-directory: ./tee-worker/client-api
+        run: pnpm run build
+
+      - name: Tee-worker ts-tests install npm deps
+        working-directory: ./tee-worker/ts-tests
+        run: pnpm install
+
```

Comment:
We have it already - exactly the following steps :)

Reply:
Fixed above.